lib/database/sfDatabaseManager.class.php

```diff
@@ -67,10 +67,14 @@ public function loadConfiguration()
   {
     if (!empty($_SERVER['DOCTRINE_DSN']))
     {
-      $databases = array('doctrine' => new sfDoctrineDatabase(array(
+      $parameters = array(
         'name' => 'doctrine',
         'dsn' => $_SERVER['DOCTRINE_DSN'],
-      )));
+      );
+      if (!empty($_SERVER['DOCTRINE_CONNECTION_CHARSET'])) {
+        $parameters['charset'] = $_SERVER['DOCTRINE_CONNECTION_CHARSET'];
```

Comment:
I think this should be `$parameters['encoding']`

Reply:
you're correct. I was confused